cpp/src/gandiva/function_registry_string.cc

```diff
@@ -515,8 +515,11 @@ std::vector<NativeFunction> GetStringFunctionRegistry() {
 
       NativeFunction("translate", {}, DataTypeVector{utf8(), utf8(), utf8()}, utf8(),
                      kResultNullIfNull, "translate_utf8_utf8_utf8",
-                     NativeFunction::kNeedsContext | NativeFunction::kCanReturnErrors)};
+                     NativeFunction::kNeedsContext | NativeFunction::kCanReturnErrors),
 
+      NativeFunction("repeatstr", {}, DataTypeVector{utf8(), int32()}, utf8(),
```

Comment:
repeat function is already present in gandiva..you can add an alias as repeatStr